### PR TITLE
[RFR] Automated advanced options

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

File: mta/entities/analysis_results.py [hunk 1]
```diff
@@ -57,15 +57,19 @@ def __init__(self, application, project_name):
         self.application = application
         self.project_name = project_name
 
+    def get_analysis_number(self, view, row):
+        """ Method to return only digit from row text"""
+        analysis_num = view.analysis_row(row).analysis_number.text
+        only_digits = "".join([c for c in analysis_num if c.isdigit()])
```
Comment (digitronik) [inline mta/entities/analysis_results.py:63]: I think we can use digits match regular expression here.

File: mta/widgetastic/__init__.py [hunk 2]
```diff
@@ -253,3 +254,12 @@ def fill(self, filepath):
...
         return self.browser.is_displayed(self)
+
+
+class AddButton(Button):
+    """Multiple buttons with same name are present in UI.
+       So need to specify the locator.
+    """
+
+    def __locator__(self):
+        return ".//button[text()='Cancel']/following-sibling::button"
```
Comment (digitronik) [inline mta/widgetastic/__init__.py:265]: btw its hack

**Review by digitronik — APPROVED** [review]
LGTM

File: mta/widgetastic/__init__.py (hunk 2 above)
Comment (sshveta) [reply mta/widgetastic/__init__.py:265]: Yea there two buttons with same name so I had to do this , But this UI is going to be replaced with new UI soon.

File: mta/entities/analysis_results.py (hunk 1 above)
Comment (sshveta) [reply mta/entities/analysis_results.py:63]: Hope this is okay to use

**Pull request merged**